x/ibc/04-channel/keeper/timeout.go

```diff
@@ -91,6 +86,11 @@ func (k Keeper) TimeoutPacket(
 
 	switch channel.Ordering {
 	case exported.ORDERED:
+		// check that packet has not been received
+		if nextSequenceRecv > packet.GetSequence() {
```

Comment:
This needs to be `>` not `>=` since nextSequenceRecv should always be 1 larger than last received packet sequence

Reply:
ditto comment

Reply:
This is a mistake in the spec, sorry - fixed in https://github.com/cosmos/ics/commit/a819bb56f28a2b2678ab1ec02f417bfb8c93565b